navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/instruction/defaultinstructions/DepartureInstruction.java

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.services.android.navigation.v5.instruction.defaultinstructions;
+
+import com.mapbox.services.android.navigation.v5.NavigationConstants;
+import com.mapbox.services.android.navigation.v5.instruction.Instruction;
+import com.mapbox.services.android.navigation.v5.milestone.MilestoneEventListener;
+import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress;
+
+/**
+ * Used to provide the {@link String} instruction in
+ * {@link MilestoneEventListener#onMilestoneEvent(RouteProgress, String, int)}
+ * for Departure Milestones
+ *
+ * @since 0.4.0
+ */
+public class DepartureInstruction extends Instruction {
```

Comment:
Rather than create separate files for each class, why not build these instructions directly in `mapboxNavigation` near the milestone creation.